docs/ddl-of-carbondata.md

```diff
@@ -1088,4 +1088,14 @@ Users can specify which columns to include and exclude for local dictionary gene
   TBLPROPERTIES ('BUCKETNUMBER'='4', 'BUCKETCOLUMNS'='productName')
   ```
 
+## CACHE
+
+  ```sql
+    DROP METADATA ON TABLE tableName
```

Comment:
previously in PR3078, it shows the metacache using
```
SHOW METACACHE FOR TABLE t1
```
So here, I'd recommend using 
```
DROP METACACHE FOR TABLE t1
```

Reply:
After this [comment](https://github.com/apache/carbondata/pull/3138#discussion_r261953503), I changed it to `ON TABLE` and deleted `FOR TABLE` from SHOW command also.

Reply:
OK, so the description of the PR is not updated later

Reply:
@NamanRastogi OK, but here is 'METACACHE' not 'METADATA'

Reply:
Yes, that must be a mistake on PR 3078.
@QiangCai Please update the description of PR 3078 and change `SHOW METACACHE FOR TABLE tableName` to `SHOW METACACHE ON TABLE tableName`